tests/backup/backup_helper.go

```diff
@@ -2026,3 +2028,30 @@ func MapToKeyValueString(m map[string]string) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+// LicenseConsumedCountVerify verifies the consumed license count for px-backup
+func LicenseConsumedCountVerify(ctx context.Context, OrgId string, expectedLicenseConsumedCount int64) error {
+	licenseInspectRequestObject := &api.LicenseInspectRequest{
+		OrgId: OrgId,
+	}
+	licenseCountCheck := func() (interface{}, bool, error) {
+		licenseInspectResponse, err := Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+		if err != nil {
+			return "", false, err
+		}
+		licenseResponseInfoFeatureInfo := licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
+		if licenseResponseInfoFeatureInfo[0].Consumed == expectedLicenseConsumedCount {
+			return "", false, nil
+		}
+		return "", true, nil
```

Comment:
Return some error here otherwise the `task.DoRetryWithTimeout` will not retry again
You can send a `Errorf` with the expected and actual count

Reply:
done